amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -2597,6 +2632,27 @@ RpptDataType getRpptDataType(vx_enum vxDataType) {
     }
 }
 
+RpptLayout getRpptLayout(vxTensorLayout layout) {
+    switch(layout) {
```

Comment:
for these kinds of const mapping, it is better to use std::map which is const and initialized before

Reply:
Removed the switch statement and used const std::map for layout mapping